src/languages/lsl.js

```diff
@@ -0,0 +1,133 @@
+/*
+    Language: Linden Scripting Language
+    Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+    Author: Builder's Brewery <buildersbrewery@gmail.com>
+    Category: common, scripting
+*/
+
+function(hljs) {
+    return {
+        case_insensitive: false,
+        aliases: ['lsl'],
```

Comment:
No need to mention alias which equals the file names — it works as a canonical name already.